<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lobby</class>
 <widget class="QMainWindow" name="Lobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background:url(:/LoginForm/pattern.png);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
background:url(:/background/pattern.png);
}</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>120</y>
      <width>511</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {background:transparent;
 background-color: rgba(0, 51, 102, 0.9); /* Albastru închis cu transparență mai mare */
    border-radius: 15px; /* Colțuri rotunjite */}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: rgb(255, 255, 255); /* Text alb pentru contrast maxim */
    font-size: 18px; /* Mărimea fontului pentru lizibilitate îmbunătățită */
    font-weight: bold; /* Font îngroșat pentru a ieși în evidență */
  
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot;font-family:'Times New Roman'; font-size:15pt; font-weight:700;  color:#ffffff;&quot;&gt;Room Owner&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>320</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: rgb(255, 255, 255); /* Text alb pentru contrast maxim */
    font-size: 18px; /* Mărimea fontului pentru lizibilitate îmbunătățită */
    font-weight: bold; /* Font îngroșat pentru a ieși în evidență */
  
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot;font-family:'Times New Roman'; font-size:15pt; font-weight:700;  color:#ffffff;&quot;&gt;Room ID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="roomIdField">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>320</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	background:transparent;
    background-color: rgba(255, 255, 255, 0.9); /* Alb cu transparență pentru a lăsa să se întrevadă fundalul */
    color: rgb(0, 0, 0); /* Textul în culoarea neagră pentru citire ușoară */
    border: 1px solid rgb(255, 255, 255); /* Bordură albă clară */
    border-radius: 5px; /* Margini rotunjite */
    padding: 5px; /* Spațiere internă pentru text */
    font-size: 16px; /* Mărimea fontului adecvată */
   
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>170</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
	background:transparent;
    background-color: rgba(255, 255, 255, 0.9); /* Alb cu transparență pentru a lăsa să se întrevadă fundalul */
    color: rgb(0, 0, 0); /* Textul în culoarea neagră pentru citire ușoară */
    border: 1px solid rgb(255, 255, 255); /* Bordură albă clară */
    border-radius: 5px; /* Margini rotunjite */
    padding: 5px; /* Spațiere internă pentru text */
    font-size: 16px; /* Mărimea fontului adecvată */  
}

QAbstractItemView{
 background:transparent;
}</string>
     </property>
     <item>
      <property name="text">
       <string>1 Player</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 Players</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3 Players</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 Players</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: rgb(255, 255, 255); /* Text alb pentru contrast maxim */
    font-size: 18px; /* Mărimea fontului pentru lizibilitate îmbunătățită */
    font-weight: bold; /* Font îngroșat pentru a ieși în evidență */
  
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot;font-family:'Times New Roman'; font-size:15pt; font-weight:700;  color:#ffffff;&quot;&gt;Number of players&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="roomOwnerField">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	background:transparent;
    background-color: rgba(255, 255, 255, 0.9); /* Alb cu transparență pentru a lăsa să se întrevadă fundalul */
    color: rgb(0, 0, 0); /* Textul în culoarea neagră pentru citire ușoară */
    border: 1px solid rgb(255, 255, 255); /* Bordură albă clară */
    border-radius: 5px; /* Margini rotunjite */
    padding: 5px; /* Spațiere internă pentru text */
    font-size: 16px; /* Mărimea fontului adecvată */
   
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>511</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {background:transparent;
 background-color: rgba(0, 51, 102, 0.9); /* Albastru închis cu transparență mai mare */
    border-radius: 15px; /* Colțuri rotunjite */}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: rgb(255, 255, 255); /* Text alb pentru contrast maxim */
    font-size: 18px; /* Mărimea fontului pentru lizibilitate îmbunătățită */
    font-weight: bold; /* Font îngroșat pentru a ieși în evidență */
  
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot;font-family:'Times New Roman'; font-size:15pt; font-weight:700;  color:#ffffff;&quot;&gt;
&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: rgb(255, 255, 255); /* Text alb pentru contrast maxim */
    font-size: 18px; /* Mărimea fontului pentru lizibilitate îmbunătățită */
    font-weight: bold; /* Font îngroșat pentru a ieși în evidență */
  
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot;font-family:'Times New Roman'; font-size:15pt; font-weight:700;  color:#ffffff;&quot;&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>220</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: rgb(255, 255, 255); /* Text alb pentru contrast maxim */
    font-size: 18px; /* Mărimea fontului pentru lizibilitate îmbunătățită */
    font-weight: bold; /* Font îngroșat pentru a ieși în evidență */
  
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot;font-family:'Times New Roman'; font-size:15pt; font-weight:700;  color:#ffffff;&quot;&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>300</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: rgb(255, 255, 255); /* Text alb pentru contrast maxim */
    font-size: 18px; /* Mărimea fontului pentru lizibilitate îmbunătățită */
    font-weight: bold; /* Font îngroșat pentru a ieși în evidență */
  
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot;font-family:'Times New Roman'; font-size:15pt; font-weight:700;  color:#ffffff;&quot;&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
